datasets/us/fincen_special_measures/us_fincen_special_measures.yml

```diff
@@ -0,0 +1,41 @@
+title: "FinCEN 311 and 9714 Special Measures"
```

Comment:
could you add data assertions for this? https://zavod.opensanctions.org/best_practices/merge_checklist/